system/t06_publish/drop.py

```diff
@@ -39,9 +40,33 @@ class PublishDrop2Test(BaseTest):
     def check(self):
         super(PublishDrop2Test, self).check()
 
-        self.check_not_exists('public/ppa/smira/dists/')
-        self.check_not_exists('public/ppa/smira/pool/')
-        self.check_exists('public/ppa/smira/')
+        self.check_not_exists('public/ppa/')
+        self.check_exists('public/')
+
+
+class PublishDrop21Test(BaseTest):
+    """
+    publish drop: under prefix
+    if some publishes have common part of the prefix, only empty parts are cleared
+    (https://github.com/aptly-dev/aptly/pull/1381)
```

Comment:
no links here, please describe the test behavior like in the other tests

Reply:
fixed

Reply:
I still see the link...

Reply:
weird... seems git amend set me )

really fixed now